crates/nu-parser/src/parse_keywords.rs

```diff
@@ -167,6 +171,7 @@ pub fn parse_def_predecl(working_set: &mut StateWorkingSet, spans: &[Span]) {
 
     // Now, pos should point at the command name.
     let name_pos = pos;
+    pos += 1;
```

Comment:
This function iterates through spans by index. The original implementation continued iterating `pos` to find the signature. I changed it to use `position` instead. But the resulting index is relative to its slice. Here I increment `pos` so it can be added to the offset a few lines below.